managed/resource_cluster.go

```diff
@@ -481,6 +606,69 @@ func validateCredentials(credentials Credentials) bool {
 
 }
 
+func createCmkSpec(plan Cluster) (cmkSpec *openapiclient.CMKSpec, cmkSpecOK bool, errorMessage string) {
```

Comment:
Instead of a `(bool, string)`, look into the other places where we return an `Error` object. That is the `go way`.

Reply:
Done